<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideTextAnimation</class>
 <widget class="QWidget" name="SlideTextAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Enter Text:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="goButton">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="4" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Duration</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="durationSpinBox">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>100000000</number>
           </property>
           <property name="value">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Easing Curve</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="easingCurveCombo">
           <item>
            <property name="text">
             <string>Linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InQuad</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutQuad</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutQuad</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInQuad</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InCubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutCubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutCubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInCubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InQuart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutQuart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutQuart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInQuart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InQuint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutQuint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutQuint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInQuint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InSine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutSine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutSine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInSine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InExpo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutExpo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutExpo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInExpo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InCirc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutCirc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutCirc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInCirc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InElastic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutElastic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutElastic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInElastic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InBack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutBack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutBack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInBack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InBounce</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutBounce</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InOutBounce</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutInBounce</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InCurve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OutCurve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SineCurve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CosineCurve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BezierSpline</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TCBSpline</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
